notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -226,7 +226,7 @@
       "source": [
```

Comment:
<p>Add some description the purpose of these two datasets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>